lib/target.py

```diff
@@ -109,16 +109,14 @@ def runTasks(self):
             elif isinstance(one_task, task.NessusTask):
                 nessus_results = one_task.runNessusScan()
                 if nessus_results:
-                    epoch_cdate = nessus_results.histories()[0].creation_date
+                    self.resultsdict.update({"nessus": True})
```

Comment:
I feel like we're asking "Target" object to do too much here.  I  think we should consider another means to invoke specific scans without burying too much scanner type specific logic in here.

Reply:
I feel like the purpose of having a class Structure is to help inform boundaries of where logic should live.   If we're burying a bunch  of task specific logic in Target, it just feels like we're blurring those lines too much.